routes/auth.js

```diff
@@ -30,9 +30,19 @@ router.get('/register', (req, res) => {
 
 // handle sign up logic
 router.post('/register', (req, res) => {
+  // get data from form and add to user array
```

Comment:
The user is not an array. It's an object. So probably adding new user object to users collection (array) in the database.